crates/core/src/lib.rs

```diff
@@ -37,6 +38,104 @@ impl agent::Verifier for Ed25519Verifier {
     }
 }
 
+struct Ed25519LocalAgentInner {
+    cb: Option<Arc<dyn Fn() + 'static + Send + Sync>>,
+    cur: DhtArc,
+    tgt: DhtArc,
+}
+
+/// A default in-memory [kitsune2_api::agent::LocalAgent] based on ed25519.
+pub struct Ed25519LocalAgent {
```

Comment:
For me, this is a minimum viable implementation that someone not using lair might choose to use in a production application. That's why I put it in core. But happy to consider alternatives : )

Reply:
(there is a `from_seed` constructor, so you can use your own seed persisted outside the scope of kitsune)

Reply:
Gotcha, yeah that's good then. I'd rather use something like this than Lair in Wind Tunnel tests for example, just because it's simpler and less to go wrong and update. Now I know what this is for, I think it makes sense for it to be here